dotcom-rendering/src/components/LoopVideoPlayer.tsx

```diff
@@ -137,8 +137,8 @@ export const LoopVideoPlayer = forwardRef(
 					id={loopVideoId}
 					css={videoStyles(width, height)}
 					ref={ref}
-					role="button"
```

Comment:
I initially added this as the video has click functionality - it pauses and plays the video. However,  I think setting the role to button is more unhelpful than helpful to a user, given that the primary function of the video is not a button.